importer/tests.py

```
@@ -1,3 +1,518 @@
+        """Test that API returns empty directories when root directory doesn't exist."""
+        # Mock /input not being a directory
+        mock_input_path = MagicMock()
+        mock_input_path.is_dir.return_value = False
+        mock_input_path.exists.return_value = False
+
+        mock_path_class.return_value = mock_input_path
+        mock_path_class.home.return_value = "/home/testuser"
+
+        response = self.client.get("/api/directories/", follow=True)
+
+        self.assertEqual(response.status_code, 404)
```

Ruff (0.15.0)

[warning] Use a regular `assert` instead of unittest-style `assertEqual`

Replace `assertEqual(...)` with `assert ...`

(PT009)